app/components/trainees/confirmation/programme_details/view.rb

```diff
@@ -0,0 +1,34 @@
+module Trainees
+  module Confirmation
+    module ProgrammeDetails
+      class View < GovukComponent::Base
+        include SanitizeHelper
```

Comment:
I think we can remove this because we don't seem to be using it

Reply:
Ah yeah, my bad, just copied it across!